### ISSUE TITLE
[Terokkar] [World PVP] Can't cap Spirit Towers / No progress bar for capping

### ISSUE THREAD

**Issue by Mseecher** [Nextcalibur/Bugtracker#592]
The spirit towers in Terrokar can't be capped, and progress bar for capping the tower doesn't show up. (Reference attached screenshot). Additionally, I tested to make sure that the tower can't be capped (originally though maybe progress bar was just not visible, but tower could still be capped) however I sat in the tower's range for ~10 minutes, which would be well enough to cap it normally, and it still didn't cap. Timestamp 4:28am server time, on 8-Aug-2020.

<img width="817" alt="Screenshot" src="https://user-images.githubusercontent.com/69408474/89723551-0b9fde80-d9c6-11ea-8594-198bac02832e.png">

**Comment by tonine-entro** [comment]
Tested and unable to reproduce 
![image](https://user-images.githubusercontent.com/67378152/90275193-83b34c00-de7f-11ea-83fb-205034a2be1d.png)

**Comment by MihawkEntropius** [comment]
Re opened the bug report, it's possible that under certain conditions it is possible to cap.

**Comment by tonine-entro** [comment]
During testing inevitably encountered a scenario where the values of the flag didn't update because of which the faction flag was yet under display and capturing the tower was not being performed.

After a server restart and clear cache, seems to me the issue of the flag holding the value of the previous faction was reset and we were able to control the towers again as either faction.

Safe to assume the issue arose because of value of the flags stored in the server memory and following restart it got cleared.

During further testing, seized all the towers as alliance and after 6 hrs. The buff was removed and all towers went back to their neutral state, which allowed them to be captured them once again.

The below image is captured after the 6hrs passed and towers went back to their neutral state.
![image](https://user-images.githubusercontent.com/67378152/115899925-019f0c80-a47d-11eb-92a6-e87d1aa8fa23.png)
